src/button/index.tsx

```diff
@@ -106,7 +106,7 @@ export default class Button extends PureComponent <ButtonProps, ButtonState> {
       // Actions
       onClick, onBlur, onFocus, onDoneAnimationEnd, focus,
       // Extend case of the button for the expand component
-      ...attrs,
+      ...attrs /* tslint:disable-line */
```

Comment:
why does this line requires the linter to be disabled?

Reply:
I agree, just put the comma back at the end of the line?